packages/components/src/elevation/test/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import Elevation from '../index';
+
+describe( 'props', () => {
+	test( 'should render correctly', () => {
+		const { container } = render( <Elevation /> );
+
+		expect( container.firstChild ).toMatchSnapshot();
+	} );
+
+	test( 'should render isInteractive', () => {
+		const { container } = render( <Elevation isInteractive /> );
+
+		expect( container.firstChild ).toMatchSnapshot();
```

Comment:
More snapshot tests 😱

Reply:
I know :(. Both `Elevation` and `Surface` are basically PURE css things.

This raises an interesting question. How should we handle unit testing for these low level primitives that are mostly CSS things?

It could get very verbose to check for individual css props 🤔  - for example, complex box shadows.

Reply:
I honestly have no idea. Checking for those things seems just as brittle as a snapshot test.